main.py

```diff
@@ -104,24 +112,24 @@ def add_token(ticker, status, email, asset_id, description='Description', link='
           'ArcaneLabs Platform System Token is utilized in CryptoBrokers technology solutions ',
           'https://rcanelabs.com', acl, 0)
 add_token('BCH', 2, 'support@turtlenetwork.eu', 'Fr2kNhe7XR3E16W7Mfh7NhNcsQWLXx3hSLjoFgpbFsNj',
-          'Bitcoin Cash proxy token for the gateway operated by Black Turtle BVBA.', 'https://turtlenetwork.eu', '', 0)
+          'Bitcoin Cash proxy token for the gateway operated by Black Turtle BVBA.', 'https://turtlenetwork.eu', bch, 0)
 add_token('BTC', 2, 'support@turtlenetwork.eu', '5Asy9P3xjcvBAgbeyiitZhBRJZJ2TPGSZJz9ihDTnB3d',
           'Bitcoin proxy token for the gateway operated by Black Turtle BVBA.', 'https://turtlenetwork.eu', '')
 add_token('DASH', 2, 'support@turtlenetwork.eu', 'A62sRG58HFbWUNvFoEEjX4U3txXKcLm11MXWWS429qpN',
-          'Dash proxy token for the gateway operated by Black Turtle BVBA.', 'https://turtlenetwork.eu', '')
+          'Dash proxy token for the gateway operated by Black Turtle BVBA.', 'https://turtlenetwork.eu', dash)
 add_token('DOGE', 2, 'support@turtlenetwork.eu', 'HDeemVktm2Z68RMkyA7AexhpaCqot1By7adBzaN9j5Xg',
-          'Doge proxy token for the gateway operated by Black Turtle BVBA.', 'https://turtlenetwork.eu', '')
+          'Doge proxy token for the gateway operated by Black Turtle BVBA.', 'https://turtlenetwork.eu', doge)
 add_token('ETH', 2, 'support@turtlenetwork.eu', '6Mh41byVWPg8JVCfuwG5CAPCh9Q7gnuaAVxjDfVNDmcD',
           'ETH proxy token for the gateway operated by Black Turtle BVBA.', 'https://turtlenetwork.eu', '')
 add_token('ETHO', 2, 'support@kgbconcepts.network', 'GzHRyYtdwvaGSkUC4i8d3Xzmsz9aXBWdrpMNszi6bcvR', 'proxy',
           'https://ether1.org', etho, 0)
 add_token('HN', 2, '------', '3GvqjyJFBe1fpiYnGsmiZ1YJTkYiRktQ86M2KMzcTb2s', 'Hellenic Node project',
-          'https://www.hellenicnode.eu', '', 0)
+          'https://www.hellenicnode.eu', HN, 0)
 add_token('LTC', 2, 'support@turtlenetwork.eu', '3vB9hXHTCYbPiQNuyxCQgXF6AvFg51ozGKL9QkwoCwaS',
           'LTC proxy token for the gateway operated by Black Turtle BVBA.', 'https://turtlenetwork.eu')
-add_token('MN', 2, '------', 'DfutD8DdUhDphHoaMds2RQhw7oCmsf7W41s2zR5ZDq9F')
```

Comment:
The order off arguments and the value is according to this:
add_token(ticker, status, email, asset_id, description='Description', link='example.com', logo='', version=0):